plugins/saveactions.c

```diff
@@ -548,18 +1135,36 @@ static void configure_response_cb(GtkDialog *dialog, gint response, G_GNUC_UNUSE
 		g_key_file_set_boolean(config, "autosave", "save_all", autosave_save_all);
 		g_key_file_set_integer(config, "autosave", "interval", autosave_interval);
 
-		if (instantsave_default_ft != NULL)
-			g_key_file_set_string(config, "instantsave", "default_ft", instantsave_default_ft);
+		g_key_file_set_integer(config, "backupcopy", "dir_levels", backupcopy_dir_levels);
+		g_key_file_set_string(config, "backupcopy", "time_fmt", text_time);
+		SETPTR(backupcopy_time_fmt, g_strdup(text_time));
+		if (enable_backupcopy)
+		{
+			if (!EMPTY(backupcopy_text_dir) && store_target_directory(
+					backupcopy_text_dir, &backupcopy_backup_dir))
+			{
+				g_key_file_set_string(config, "backupcopy", "backup_dir", backupcopy_text_dir);
+			}
+			else
+			{
+				dialogs_show_msgbox(GTK_MESSAGE_ERROR,
+						_("Backup directory does not exist or is not writable."));
+			}
+		}
+
+		if (untitled_doc_default_ft != NULL)
+			g_key_file_set_string(config, "untitled_document_save", "default_ft", untitled_doc_default_ft);
+
 		if (enable_instantsave)
 		{
 			if (EMPTY(instantsave_text_dir))
 			{
-				g_key_file_set_string(config, "instantsave", "target_dir", "");
+				g_key_file_set_string(config, "untitled_document_save", "instantsave_target_dir", "");
```

Comment:
I think the whole plugin would deserve to define section and config names as macros - this is a bit error-prone.

Probably not necessary as part of this PR, I might open a separate PR with this afterwards.